### PR TITLE
Gh 205 logging

### PR DESCRIPTION
**Pull request by go159lab**
Add logging to the gaas-rest using SLF4J with stable Log4j2.

### CODE REVIEW

File: gaffer-as-a-service/pom.xml [hunk 1]
```diff
@@ -40,7 +40,6 @@
         <kubernetes.version>11.0.1</kubernetes.version>
         <fasterxml.jackson.version>2.11.2</fasterxml.jackson.version>
         <javax.xml.bind.version>2.3.0</javax.xml.bind.version>
-        <slf4j.api.version>1.7.25</slf4j.api.version>
```
Comment (t92549) [inline gaffer-as-a-service/pom.xml:43]: I thought you needed slf4j AND log4j dependencies?

Comment (t92549) [reply gaffer-as-a-service/pom.xml:43]: This is also creating a merge conflict

**Review by t92549 — CHANGES_REQUESTED** [review]

File: gaffer-as-a-service/pom.xml (hunk 1 above)
Comment (go159lab) [reply gaffer-as-a-service/pom.xml]: Updated the log4j and slf4j dependencies and resolved merge conflict

File: gaffer-as-a-service/pom.xml [hunk 2]
```diff
@@ -41,6 +41,7 @@
         <fasterxml.jackson.version>2.11.2</fasterxml.jackson.version>
         <javax.xml.bind.version>2.3.0</javax.xml.bind.version>
         <slf4j.api.version>1.7.32</slf4j.api.version>
+        <log4j.version>1.2.17</log4j.version>
```
Comment (t92549) [inline gaffer-as-a-service/pom.xml:44]: You did make this log4j2 and added a log4j2.properties but now the version is back to 1.2.17

**Review by t92549 — CHANGES_REQUESTED** [review]

File: gaffer-as-a-service/pom.xml (hunk 2 above)
Comment (go159lab) [reply gaffer-as-a-service/pom.xml]: Please review @t92549 
As I updated the pom.xml with the log4j2 stable version and dependencies. Tested/debuged the code and the logging works fine.

**Review by t92549 — APPROVED** [review]

**Pull request merged**